.github/workflows/ci.yml

```diff
@@ -110,7 +118,7 @@ jobs:
         if: matrix.node != 14
         run: pnpm install --frozen-lockfile --prefer-offline --ignore-scripts
       - name: install for node14
-        if: matrix.node == 14
+        if: matrix.node == 14 || matrix.svelte == 3
```

Comment:
If svelte3 run finishes first, the pnpm cache saved might not work with svelte4. We may have to provide a custom cache key unless node action somehow figures it out. (eg by using a hash of the lockfile?)

Reply:
It doesn't seem like `setup-node` supports providing a custom cache key, but also I don't think mixing Svelte 3 and 4 would be an issue since it's only a cache with the same shape.

Worst case might be pnpm needing to download Svelte 3 and 4 from the network inconsistently, and that might be better than disabling cache for Svelte 3 altogether.

Reply:
hmm we can try it but I fear the cache gets busted in a way that is not good.
It would be better to check how to have separate caches for 3 and 4 ci

Reply:
The test failure archive needs svelte dimension in the name too